packages/@angular/cli/commands/update.ts

```diff
@@ -39,16 +39,13 @@ export default class UpdateCommand extends SchematicCommand {
   }
 
   public async run(options: UpdateOptions) {
-
-    const schematicRunOptions = {
+    return this.runSchematic({
       collectionName: this.collectionName,
       schematicName: this.schematicName,
       schematicOptions: options,
       dryRun: options.dryRun,
       force: false,
-      workingDir: this.project.root,
```

Comment:
Why was `workingDir` removed? It doesn't look related to this fix.

Reply:
It's not actually used anywhere in the function.